addon/templates/components/common/bs-carousel.hbs

```diff
@@ -0,0 +1,23 @@
+{{#if showIndicators}}
+  <ol class='carousel-indicators'>
+    {{#each indicators as |indicator index|}}
+      <li class='{{if (bs-eq currentIndex index) 'active'}}' onclick={{action 'toSlide' index}} role='button'></li>
+    {{/each}}
+  </ol>
+{{/if}}
+
+<div class='carousel-inner' role='listbox'>
+  {{yield (hash slide=(component 'bs-carousel/slide'))}}
+</div>
+
+{{#if showControls}}
+  <a class="{{prevControlClassName}}" href="#{{elementId}}" onclick={{action 'toPrevSlide'}} role="button">
```

Comment:
When you remove `onclick=` here, i.e. use the action helper in element space, it will automatically call `preventDefault`. which will fix the issue reported here: https://github.com/kaliber5/ember-bootstrap/pull/417#issuecomment-325550494

Or you need to call `event.preventDefault()` explicitly in the called actions!

Reply:
Nice!